examples/blockly-parcel/src/generator.js

```diff
@@ -15,17 +15,86 @@ import 'blockly/blocks';
 import 'blockly/python';
 import * as En from 'blockly/msg/en';
 
+const toolbox = {
+  "kind": "flyoutToolbox",
+  "contents": [
+    {
+      "kind": "block",
+      "type": "controls_ifelse"
+    },
```

Comment:
Please check [the Google JavaScript Style Guide](https://google.github.io/styleguide/jsguide.html) and make sure the code is formatted appropriately.  In particular:

- Use `''` rather than `""` for string literals.
- Do not quote object literal keys when not required.
- Include trailing commas in block-formatted lists.

```suggestion
const toolbox = {
  kind: 'flyoutToolbox',
  contents: [
    {
      kind: 'block',
      type: 'controls_ifelse',
    },
```
(etc.)